rust-toolchain.toml

```
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2024-03-10" # rustc 1.78.0
+channel = "stable-2024-04-09" # rustc 1.77.2
```

Did you have any issue having:
```suggestion
channel = "1.77.2"
```
?